webroot/src/components/StateUpload/StateUpload.ts

```diff
@@ -68,18 +73,8 @@ class StateUpload extends mixins(MixinForm) {
         return compactsList;
     }
 
-    get compactOptions(): any {
-        const valueOptions = [
-            { value: '', name: computed(() => this.$t('common.selectOption')) }
-        ].concat(this.compacts.map((compact) => ({
-            value: compact.key, name: compact.name
-        })));
-
-        return valueOptions;
-    }
-
     get compactId(): string {
-        return this.formData.compact?.value || '';
+        return this.userStore.currentCompact || Compact.ASLP;
```

Comment:
If `currentCompact` is null and the user does not have access to `ASLP`, what is the consequence of this default?

Reply:
For the next set of demos & user research, all users are supposed to be created under ASLP. If for some reason they don't have that access, then the server fetches will fail and the user will see the network error UI.